modules/v1_14_R1/src/main/java/net/countercraft/movecraft/compat/v1_14_R1/IWorldHandler.java

```diff
@@ -103,16 +105,36 @@ public void rotateCraft(@NotNull Craft craft, @NotNull MovecraftLocation originP
         //*         Step two: Get the tiles         *
         //*******************************************
         WorldServer nativeWorld = ((CraftWorld) craft.getWorld()).getHandle();
-        List<TileHolder> tiles = new ArrayList<>();
+        List<TileEntity> tiles = new ArrayList<>();
+        List<NextTickListEntry> tileTicks = new ArrayList<>();
+        HitBox testBox = craft.getHitBox();
+        BlockPosition position1 = new BlockPosition(testBox.getMidPoint().getX() - testBox.getZLength()/2 - 2, 0, testBox.getMidPoint().getZ() - testBox.getXLength()/2 - 2);
+        BlockPosition position2 = new BlockPosition(testBox.getMidPoint().getX() + testBox.getZLength()/2 + 2, 0, testBox.getMidPoint().getZ() + testBox.getXLength()/2 + 2);
```

Comment:
Use `testBox.get[Min/max][X/Y/Z]()`

Reply:
This is done intentionally; the plugin has to search a square centered on the craft's position to consistently find all the tile ticks, I've found. No idea why this is the case, though.

Reply:
(this is only the case when rotating, translation can use the normal hitbox)